GettingStarted.md

```diff
@@ -0,0 +1,114 @@
+# SailfishOS Chum - Getting Started with OBS
+
+This document summarises the proces of building your first package on 
+OBS.  It only skims the surface of OBS and is not a manual.
+
+To start building your packages on OBS you will require an account.
+Follow the process in the README and contact 'lbt' for an account to 
+be set up.
+
+When your account is set up, you will have a "Home Project".  This is where
+you can start creating packages.  The URL for your home project will be:
+https://build.sailfishos.org/project/show/home:<username>
+
+## Creating a package on OBS
+
+Projects contain packages which are built against target repositories
+(sailfish versions).  To create your first package, perform the following
+steps:
+
+1. Click "Create Package"
+2. On the new package page, give the package a name and click "Save Changes"
+   * eg harbour-mypackage
+3. You now need to add files to the package to allow OBS to get the source and
+build it.  Create a file on your computer called `_service` with the following content:
+```
+<services>
+    <service name="tar_git">
+    <param name="url">URL TO PROJECT</param>
+    <param name="branch">master</param>
+    <param name="revision"></param>
+    </service>
+</services>`
+```
```

Comment:
See my other comment, the same applies here.
````suggestion
    ```
    <services>
        <service name="tar_git">
        <param name="url">URL TO PROJECT</param>
        <param name="branch">master</param>
        <param name="revision"></param>
        </service>
    </services>`
    ```
````